simulation/adx_gyro_accelerometer/build.gradle

```diff
@@ -11,6 +11,17 @@ if (!project.hasProperty('onlyAthena')) {
 
 
     model {
+        dependencyConfigs {
+            wpiutil(DependencyConfig) {
+                groupId = 'edu.wpi.first.wpiutil'
+                artifactId = 'wpiutil-cpp'
+                headerClassifier = 'headers'
+                ext = 'zip'
+                version = '3.+'
+                sharedConfigs = [ halsim_adx_gyro_accelerometer: [] ]
+                staticConfigs = [ halsim_adx_gyro_accelerometerStaticDeps: [] ]
```

Comment:
You don't need the halsim_adx_gyro_accelerometerStaticDeps. You can just leave staticConfigs as an empty array.